lib/init.coffee

```diff
@@ -4,6 +4,11 @@ module.exports =
       type: 'string'
       title: 'the path to SwiftLint\'s configuration file that relative to project or absolute'
       default: '.swiftlint.yml'
+    additionalOptions:
+      type: 'string'
+      title: 'additional options to SwiftLint\'s `lint` command
```

Comment:
This should be a `description`, not `title` (yes, all the other options are wrong... 😛).